src/components/ItemsList.js

```diff
@@ -19,7 +21,12 @@ const ItemsList = () => {
         {loading && <p>Loading...</p>}
         {error && <p>An error has occured...</p>}
         {shoppingList && !shoppingList.length && (
-          <p>You haven't created a shopping list yet...</p>
+          <div>
+            <p>You haven't created a shopping list yet...</p>
+            <Link to="/addItem" className="add-item">
+              <button>Add Item</button>
+            </Link>
```

Comment:
I have just noticed you are rendering  `button` as a child of `Link` component. Essentially `Link` renders an `a` tag.  When you check in the browser console you will notice something like:
```html
<a class="add-item" href="/addItem">
   <button>Add Item</button>
</a>
```
This, according to the [html spec.](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element), is not valid markup since `button` is an interactive element as stated in the specification.
>  There must be no interactive content descendant, `a` element descendant, or descendant with the `tabindex` attribute specified.

You can also check out [this](https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5) stackoverlfow question. Similarly running a markup which has a `button` element as a descendant of `a` element in the [HTML validator](https://validator.w3.org/nu/#textarea) will emit the following error:

> The element `button` must not appear as a descendant of the `a` element.

It seems you cannot nest  `a` inside `button` either. 
I would suggest the following:
1. If you want to display a button, then use `click` event listener to redirect the user to `additem` page. 
2. You can also have a `span` inside `Link` which you can style to look like a `button` instead of nesting a `button` inside `Link` like you did.

Reply:
@nibble0101 Ah great thanks - that's a good observation, I didn't know this! And thanks for the suggestions - I think the best thing to do would be what you suggest as option (1) - display a button and add an event listener & useHistory to redirect. @Amaka202 what do you think?

Reply:
Maybe we could also work with useHistory again here since there seem to be a few advantages as Luka and Muhammad told us?

Reply:
Ok thank you Joseph. option (1) so we continue to use props.history

Reply:
the redirect has been updated to use history.push [here](https://github.com/the-collab-lab/tcl-19-smart-shopping-list/commit/9ff3b04a45c5bc315cbe9de4d2658a487746eb40)

Reply:
Thanks @Amaka202 . This looks good to me. Good job, team.